platforms/nuttx/src/px4/nxp/imx9/version/board_mcu_version.c

```diff
@@ -70,8 +70,8 @@
 #  define min(a,b) (((a) < (b)) ? (a) : (b))
 #endif
 
-static unsigned hw_version = 0;
-static unsigned hw_revision = 0;
+static unsigned hw_version = BOARD_DEFAULT_VER;
```

Comment:
Unrelated change

Reply:
It's kind of related. The version numbers are used in manifest parsing (selecting the correct options for the HW in guestion). The manifest parsing /version numbering could have been put into an own PR of course. But it was also needed in order to enable PX4IO, existence of which is read from the manifest (based on hw version)....

So I just decided to mash everything in this same PR, sorry for that and thanks for the comment :)